plugins/platform-ui/src/components/internal/ThemeSelector.svelte

```diff
@@ -5,21 +5,34 @@
   const { setTheme } = getContext('theme')
   function changeTheme(event: any): void {
     const sT = setTheme(event.srcElement.innerText)
+    window.removeEventListener('click', handler)
   }
   const themes = [..._themes]
 
+  function handler(event: MouseEvent) {
+    const cl = event.target.classList[0]
+    if ((cl !== 'item') && (cl !== 'menu')) {
+      hidden = !hidden
+      window.removeEventListener('click', handler)
+    }
+  }
+  
   let hidden = false
   function toggleMenu (): void {
     hidden = !hidden
+    if (hidden) {
+      window.addEventListener('click', handler)
```

Comment:
Why do you need all those `eventListeners`? Please use svelte mechanisms to work with events